cumulus/tasks/discover-pdrs/package.json

```diff
@@ -38,6 +38,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@cumulus/common": "^1.0.0-1.0.0-beta.9.1",
+    "@cumulus/cumulus-message-adapter-js": "0.0.1-beta.2",
```

Comment:
Could you add `~` to the version number here, so it automatically gets, the latest version?

Reply:
Fixed in 5c4cbe5.